src/home_robot/home_robot/utils/geometry/angles.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+
+PI2 = 2 * np.pi
+
+
+def angle_difference(angle1: float, angle2: float):
+    """angle difference"""
+    angle1 = angle1 % PI2
+    angle2 = angle2 % PI2
+    return np.abs(angle1 - angle2)
```

Comment:
Something like this may be more unified as it will handle cases (-PI2 + 0.01; PI2 - 0.01) and pi vs -pi ambiguity.
```suggestion
    diff = np.abs(angle1 - angle2) % PI2
    return min(diff, TWO_PI - diff)
```